Source: @pullmoll
Upstream: no
Reason: work around a bug in binutils 2.34 w/ the sysexit instruction

--- a/arch/x86/kernel/Makefile	2020-02-05 14:03:47.000000000 +0100
+++ b/arch/x86/kernel/Makefile	2020-02-15 23:07:43.816785356 +0100
@@ -21,6 +21,7 @@
 KASAN_SANITIZE_dumpstack_$(BITS).o := n
 
 CFLAGS_irq.o := -I$(src)/../include/asm/trace
+CFLAGS_paravirt_patch_64.o := -Wa,-mintel64
 
 obj-y			:= process_$(BITS).o signal.o
 obj-$(CONFIG_COMPAT)	+= signal_compat.o
